homeassistant/components/google_assistant/trait.py

```diff
@@ -347,59 +290,92 @@ def supported(domain, features, device_class):
         if domain != light.DOMAIN:
             return False
 
-        return features & light.SUPPORT_COLOR_TEMP
+        return (features & light.SUPPORT_COLOR_TEMP or
+                features & light.SUPPORT_COLOR)
 
     def sync_attributes(self):
         """Return color temperature attributes for a sync request."""
         attrs = self.state.attributes
-        # Max Kelvin is Min Mireds K = 1000000 / mireds
-        # Min Kevin is Max Mireds K = 1000000 / mireds
-        return {
-            'temperatureMaxK': color_util.color_temperature_mired_to_kelvin(
-                attrs.get(light.ATTR_MIN_MIREDS)),
-            'temperatureMinK': color_util.color_temperature_mired_to_kelvin(
-                attrs.get(light.ATTR_MAX_MIREDS)),
-        }
+        features = attrs.get(ATTR_SUPPORTED_FEATURES, 0)
+        response = {}
+
+        if features & light.SUPPORT_COLOR:
+            response['colorModel'] = 'rgb'
```

Comment:
Might have missed a previous conversation about this, but: Why don't we use HSV here? I mean we use HSV ourselves internally.

Reply:
So right now I just migrated it. I glanced at it briefly, but I have to confess that I don't know how to convert our HS_COLOR attribute to HSV. If it's a simple fix and you know how to convert it, it's as simple as a dict having `hue`, `saturation` and `value`: https://developers.google.com/actions/smarthome/traits/colorsetting#device-states , and then in execute also be able to deal with incoming `spectrumHSV` attribute.